api/helpers/db.js

```diff
@@ -137,17 +140,34 @@ async function _refreshSearch() {
   setTimeout(_refreshSearch, randomDelay());
 }
 
-if (!process.env.MIGRATIONS) {
-  _listUsers();
-  _listCases().then(() => console.log("cases cached"));
-  _listMethods().then(() => console.log("methods cached"));
-  _listOrganizations().then(() => console.log("organizations cached"));
-  _refreshSearch().then(() => console.log("search refreshed"));
-  db.none("UPDATE localizations SET keyvalues = ${keys} WHERE language='en'", {
-    keys: i18n_en
-  })
-    .then(() => console.log("i18n updated"))
-    .catch(error => console.error(error));
+const SUPPORTED_LANGUAGES = ["en", "de", "fr", "zh", "es"];
```

Comment:
perhaps we should store this in a shared constants file so we can have a single source of truth for supported languages. we are also using this array of languages here: https://github.com/participedia/api/blob/master/app.js#L62

Reply:
Good idea. We could maybe move the constants file into `api/helpers/data` at some point, but this is now using the same constants file as other code.